en/docs/release-notes/version-3.3.0.md

```diff
@@ -1 +1,42 @@
-# Version 3.3.0
\ No newline at end of file
+# Ballerina VSCode extension (version 3.3.0) 
+
+We are happy to announce the Ballerina VS Code plugin 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
+
+- [Visual Data Mapper](#data-mapper) - Helps you write and visualize data transformations easily
+- [GraphQL Tryit](#integrated-graphql-tryit) - Facilitates trying out the GraphQL services with the integrated client 
+- [Improved record editor](#record-editor) - Provides a better editing experience with suggestions 
+- [Project Design View](#record-editor) - Facilitates visualizing service interactions in your project
```

Comment:
Should be `Project design view`

Reply:
As this is a part of the UI, can special-case it as a pronoun and say Project Design View.

Reply:
Same as above